app/javascript/pages/Browse.vue

```diff
@@ -27,8 +27,15 @@ export default {
   },
   props: {
     contributions: {type: Array},
-    filterTypes: {type: Array},
-    initialFilters: {type: Array, default: () => []},
+    filterTypes: {type: Array, default: ()=>[]},
+    initialFilters: {
+      type: Array,
+      default: function () {
+        return [].concat(
+          ...this.filterTypes.map((fType) => fType.filters.map((filter) => filter.id))
```

Comment:
I'm using `[].concat(...array)` as a substitute for `array.flat()`

`Array.prototype.flat` is left intact by babel/webpack in our current config, and icaniuse.com lists its lack of support at above 10%. I've been aiming for 2% or 3% so far in this project